src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/UsagesOperations.cs

```diff
@@ -64,7 +64,7 @@ internal UsagesOperations(NetworkManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<IPage<Usage>>> ListWithHttpMessagesAsync(string location, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<IEnumerable<Usage>>> ListWithHttpMessagesAsync(string location, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a breaking change.  You would need to move to the next major version if you do this.